tests/e2e-evm/test/addresses.ts

```diff
@@ -0,0 +1,4 @@
+import { Address } from "viem";
+
+export const whaleAddress: Address = "0x03db6b11f47d074a532b9eb8a98ab7ada5845087";
```

Comment:
What do you think about extracting all configuration consts into one modules/file? For example, we can keep all private/public keys, chain versions, urls for connections, etc, as we already use those in multiple files among tests and this can be reused with all necessary comments.

Reply:
Ya it's a good idea, I'm just deferring for now until those configurations are used in more places.   I'd rather organize too late than too soon and the configurations are not used many places yet.

Reply:
Confirming, are these tests always run against the same environment (or spun up environment) where these address will exist?

Reply:
n/m this was answered below: https://github.com/Kava-Labs/kava/pull/2016/files#r1744001105

Reply:
Removed in a future commit